ferry.go

```diff
@@ -418,31 +450,6 @@ func (f *Ferry) Start() error {
 	// This guarentees that we are able to restart from a valid location.
 	f.StateTracker.CopyStage.UpdateLastProcessedBinlogPosition(pos)
 
-	// Loads the schema of the tables that are applicable.
-	// We need to do this at the beginning of the run as this is required
-	// in order to determine the PrimaryKey of each table as well as finding
-	// which value in the binlog event correspond to which field in the
-	// table.
-	if f.StateToResumeFrom != nil {
-		f.Tables = f.StateToResumeFrom.LastKnownTableSchemaCache
-	} else {
-		metrics.Measure("LoadTables", nil, 1.0, func() {
-			f.Tables, err = LoadTables(f.SourceDB, f.TableFilter)
-		})
-		if err != nil {
-			return err
-		}
-	}
-
-	// TODO(pushrax): handle changes to schema during copying and clean this up.
-	f.BinlogStreamer.TableSchema = f.Tables
```

Comment:
Do we not need to set the TableSchema on the BinlogStreamer in `Initialize` now?

Reply:
We do. It occurs directly in NewBinlogStreamer as opposed to within Initialize in an adhoc way.

Reply:
I see it above now. Thanks 👍